src/install.rs

```diff
@@ -136,3 +150,95 @@ fn chmod_x(file: &PathBuf) {
         error!("Failed to run chmod +x: {}", e)
     };
 }
+
+fn mac_process_zip(archive_path: String) {
+    info!("Mac zip postprocessing starting...");
+
+    // We have to do the unzipping via shell because the tool omits the app directory and doesn't set the execution flag properly
+    let archive_parent = Path::new(&archive_path).parent().unwrap().to_str().unwrap();
+    info!("  Unzipping {} to {}", archive_path, archive_parent);
+    let output = Command::new("/usr/bin/unzip")
+                            .arg(archive_path.clone())
+                            .arg("-d")
+                            .arg(archive_parent.clone())
+                            .output()
+                            .expect("Unzip failed");
+    info!("  Result of unzip: {}", output.status);
+    io::stdout().write_all(&output.stdout).unwrap();
+    io::stderr().write_all(&output.stderr).unwrap();
+
+    // delete the zip file
+    info!("  Deleting zip file {}", archive_path.clone());
+    if let Err(e) = fs::remove_file(archive_path) {
+        error!("Failed to remove archive. {}", e)
+    };
+
+    info!("Mac zip postprocessing done...");
+}
+
+fn mac_process_dmg(archive_path: String) {
+    info!("Mac dmg postprocessing starting...");
+    
+    // Mount the disk image file
+    info!("  Mounting dmg file {}", archive_path.clone());
+    let output = Command::new("/usr/bin/hdiutil")
+                            .arg("attach")
+                            .arg(archive_path.clone())
+                            .output()
+                            .expect("Mount failed");
```

Comment:
Consider using [dmg](https://lib.rs/crates/dmg) instead.

Reply:
I have the suspicion it won't work. Reason is this: I jump through the hoop of writing and executing a script, because the RUST runtime has no access to mounted drives. I spent several hours trying to find a way to get the copy command working inside rust and had to give up. Mac security won't allow it. I have granted my command line access, that's why it's working from the scripts. This is one of the things I have to mention in the install instructions.